CHANGELOG-prerelease.md

```diff
@@ -3,6 +3,7 @@
 Tag: [v1.0.0-prerelease.42](https://github.com/patternfly/patternfly-elements/releases/tag/v1.0.0-prerelease.42)
 
 - [](https://github.com/patternfly/patternfly-elements/commit/) fix: set consistent line-height for pfe-nav triggers (#773)
+- [](https://github.com/patternfly/patternfly-elements/commit/) fix: set pfe-cta default font-size to 18px
```

Comment:
@chrisdo1 should we modify this message, since the functionality is that we added a hook to override the fontsize of pfe-cta's with the priority attribute?

Reply:
oh yes. Good catch. Fixing this right now